cmd/root.go

```diff
@@ -121,6 +121,11 @@ func init() {
 		"disable colorized output")
 	viper.BindPFlag("no-color",
 		rootCmd.PersistentFlags().Lookup("no-color"))
+
+	rootCmd.PersistentFlags().Bool("skip-workspace-crud", false,
+		"Skip Workspace CRUD checks")
```

Comment:
Please change this to `Skip API calls related to Workspaces (Kong Enterprise only)`.